io/out.go

```diff
@@ -3,9 +3,82 @@ package io
 import (
 	"fmt"
 	"os"
+
+	"github.com/googleinterns/terraform-cost-estimation/resources"
+)
+
+const (
+	webBeginning = `
```

Comment:
Maybe consider moving the html template into a separate file, which is read at runtime.